WooCommerce/Classes/ViewRelated/Themes/ThemesCarouselView.swift

```diff
@@ -48,6 +58,11 @@ struct ThemesCarouselView: View {
         .task {
             await viewModel.fetchThemes()
         }
+        .sheet(isPresented: $isShowingThemesPreview) {
+            if let theme = selectedTheme {
+                ThemesPreviewView(theme: theme, onStart: { /* todo install theme */ } )
+            }
+        }
```

Comment:
This can be simplified by using the `selectedTheme` property directly:

```suggestion
        .sheet(item: $selectedTheme, content: { theme in
            ThemesPreviewView(theme: theme, onStart: { /* todo install theme */ } )
        })
```

This way, we can remove `isShowingThemesPreview`.

Reply:
This is very convenient, TIL! Also in https://github.com/woocommerce/woocommerce-ios/pull/11452/commits/232549bf80ded90b3fd469d789d86a1c55ed4fef